java/ql/test/query-tests/security/CWE-276/android/incomplete_provider_permissions/TestFull/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think you need different `AndroidManifest.xml` files for the different test cases, you can create several `<provider>` elements in the same manifest file.

Reply:
Oh true.

So would you suggest merging most all of them except for `Testbuild`?